CONTRIBUTING.md

```diff
@@ -12,10 +12,14 @@ We use <a href="http://nvie.com/posts/a-successful-git-branching-model/">Gitflow
 
 ### Pull Requests
 * Please follow the repository's for all code and documentation.
+* All pull requests must be send to `smartdevicelink/sdl_core/`, in `develop` or `master` branches.
 * All feature branches should be based on `develop` and have the format `feature/branch_name`.
-* Minor bug fixes, that is bug fixes that do not change, add, or remove any public API, should be based on `master` and have the format `hotfix/branch_name`.
+* All hotfix branches should be based on `develop` and have the format `fix/branch_name`.
```

Comment:
seems hotfix can be based on any branch (master, develop, release) since this is un-expected, fast fix. It's purpose to stabilize destination branch.